### ISSUE TITLE
[Extensions] Non-alphabetical sorting of models under Category in Components dock

### ISSUE THREAD

**Issue by leecalcote** [meshery/meshery#10873]
### Current Behavior

https://github.com/meshery/meshery/assets/7570704/0317c491-d132-4354-8236-036358a03734

### Desired Behavior
Model names should be sorted alphabetically.

### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Meshery Build and Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 UI Design For Meshery Extensions [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 📚 [Meshery Extensibility Providers](https://docs.meshery.io/extensibility/providers)
- ⌨️ [Meshery API Docs](https://docs.meshery.io/extensibility/api)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by leecalcote** [comment]
@Yashsharma1911? Or is this something that you can fix, @captain-Akshay ?

**Comment by Chidinma02** [comment]
Hi @leecalcote! 👋 I noticed this issue has been inactive for a while. Is this task still relevant?
if so,would like to pick it up for the LFX mentorship.

**Comment by ritzorama** [comment]
@Chidinma02, if you're actually interested in working on this issue, then go look and confirm.

**Comment by ritzorama** [comment]
@Chidinma02, no, the "help wanted" issue label is not on this issue. It's not an oversight. This issue is not open.
